src/mixins/filing-template-mixin.ts

```diff
@@ -176,13 +188,27 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
         }
         const createRules:CreateRulesIF = {
           valid: false,
-          rulesConfirmed: false,
+          rulesConfirmed: draftFiling.incorporationApplication.cooperative?.rulesConfirmed,
           rulesDoc: rulesDoc,
           docKey: draftFiling.incorporationApplication.cooperative?.rulesFileKey
         }
         this.setRules(createRules)
         // Set Memorandum
-        this.setMemorandum(draftFiling.incorporationApplication.memorandum)
+        let memorandumDoc:DocIF = null
+        if (draftFiling.incorporationApplication.cooperative?.memorandumFileKey) {
+          memorandumDoc = {
+            name: draftFiling.incorporationApplication.cooperative?.memorandumFileName,
+            lastModified: draftFiling.incorporationApplication.cooperative?.memorandumFileLastModified,
+            size: draftFiling.incorporationApplication.cooperative?.memorandumFileSize
+          }
+        }
+        const createMemorandum:CreateMemorandumIF = {
+          valid: false,
+          memorandumConfirmed: draftFiling.incorporationApplication.cooperative?.memorandumConfirmed,
+          memorandumDoc: memorandumDoc,
```

Comment:
Are we ok to pass `null` here when the above condition fails (line 198)?

Reply:
yeah should be fine